CATEGORIES.md

```diff
@@ -0,0 +1,79 @@
+  page with headers, margins, and multiple content flows.
+- **`text`:** Packages that transform text and strings or are focused on fonts.
+- **`scripting`:** Libraries to enhance the scripting and automation experience
+  in Typst.
+- **`integration`:** Integrations with other, third-party tools and formats.
+  This also includes packages that embed a third-party binary as a plugin.
+- **`visualization`:** Packages producing compelling visual representations of
+  data, information, and models.
+- **`utility`:** Utility packages for working with Typst and authoring packages.
+- **`fun`:** Unique uses of Typst that are not neccessarily practical, but
+  always entertaining.
+
+## Publication Categories
+- **`book`:** Long-form fiction and non-fiction books with multiple chapters.
+- **`report`:** A multi-page informational or investigative document focused on
+  a single topic. This category contains templates for tech reports, homework,
+  proposals and more.
+- **`paper`:** A scientific treatment on a research question. Usually published
+  in a journal or conference proceedings.
+- **`thesis`:** A final long-form deliverable concluding an academic degree.
+- **`poster`:** A large-scale graphics-heavy presentation of a topic. A poster
+  is intended to give its reader a first overview over a topic at a glance.
+- **`flyer`:** Graphics-heavy, small leaflets intended for massive circulation
+  and to inform or convince.
+- **`presentation`:** Slides for a projected, oral presentation.
+- **`cv`:** A résumé or curriculum vitæ presenting the author's professional
+  achievements in a compelling manner.
+- **`office`:** Staples for the day-to-day in an office, such as a letter or an
+  invoice.
+
+## Discipline Categories
+- **`education`:** Packages tailored for educators and students, including
+  exams, syllabi, and school-specific templates.
+- **`math`:** Templates for mathematical publications and tools for mathematical
+  typesetting, equations, and theorem formatting.
+- **`physics`:** Resources for visualizing physical concepts, equations, and
+  experimental data and journal templates.
+- **`chemistry`:** Packages and templates that help with chemical notation,
+  reaction diagrams, and visualization.
+- **`biology`:** Templates and tools for biological illustrations, taxonomic
+  charts, and research paper formats.
+- **`economics`:** Economic modeling tools, financial charting packages, and
+  market analysis templates.
+- **`engineering`:** Packages and templates for all engineering disciplines,
+  including but not limited to Electrial, Civil, and Mechanical engineering.
+- **`computer-science`:** Packages for algorithm visualization, code formatting,
+  and templates for Computer Science periodicals.
+- **`law`:** Legal document templates, case law formatting, and citation tools
+  for legal writing.
+- **`arts`:** Templates for the Humanities.
+- **`music`:** Packages for musical notation and composition aids.
```

Comment:
Additional ideas:
- `social-science`, or to be more precise---similar to the natural sciences:
  - `geography`
  - `linguistics`
  - `psychology`
  - `sociology`
  - `political`
- `journalism`
- `business`
- `medicine`
- `administration`

Alternatively, do something in the style of MIME types: e.g. `humanities/history`, or `applied/law`. You can use the list at https://en.wikipedia.org/wiki/Outline_of_academic_disciplines as inspiration.